l10n/exceptions/mozorg.json

```diff
@@ -49,7 +49,10 @@
           "en/footer.ftl:footer-follow-firefox",
           "en/footer.ftl:footer-follow-mozilla",
           "en/mozorg/newsletters.ftl:newsletters-beta-news"
-      ]
+      ],
+      "zh-TW": [
+          "en/mozorg/about/history.ftl:history-the-history-of-firefox-and"
+      ],  
```

Comment:
Still wrong, but I'm going to fix it directly.

We worked with JSON before, so you should be familiar. For the future, you can check also via https://jsonlint.com/

Reply:
> Still wrong, but I'm going to fix it directly.
> 
> We worked with JSON before, so you should be familiar. For the future, you can check also via https://jsonlint.com/

The last entry closing shouldn't have a comma. Comma means there is another entry to follow.